addons/im_livechat/static/src/embed/core/thread_model_patch.js

```diff
@@ -60,6 +60,7 @@ patch(Thread.prototype, {
         super.setup();
         this.chatbotTypingMessage = Record.one("Message");
         this.livechatWelcomeMessage = Record.one("Message");
+        this.operator = Record.one("Persona");
```

Comment:
Note that you can have proper typing by adding `operator = Persona` in the `@types/models.d.ts`.
See how it's done with `livechatWelcomeMessage` for example!